OneCommunityGlobal/HighestGoodNetworkApp
# XiaoW Add tooltip to new timer and cleanup old timer

# Description
This PR cleans up the old timer related code and files, and adds tooltip to buttons on the new timer. Also included goal changes for the timer behavior adjustment, details are in [PR#586](https://github.com/OneCommunityGlobal/HGNRest/pull/586#issue-1962635392)

## Related PRS (if any):
This frontend PR is related to the backend [PR#586](https://github.com/OneCommunityGlobal/HGNRest/pull/586#issue-1962635392)

## Main changes explained:
- Delete files/codes related to the old timer, including test files
- Update new timer with some of the latest changes on the old timer
- Rename NewTimer to Timer
- Small changes on TimeEntryForm.jsx, remove redundant codes
- change 'Goal for today' to 'Initial Goal'


## How to test:
1. check into current branch, check into branch [PR#586](https://github.com/OneCommunityGlobal/HGNRest/pull/586#issue-1962635392)
2. do `npm install` and `npm run start:local` to run this PR locally
3. log as any user
4. go to dashboard
5. verify timer buttons have tooltips when hovering on them
6. There are more instructions about the timer behavior changes on the [PR#586](https://github.com/OneCommunityGlobal/HGNRest/pull/586#issue-1962635392)

## Note: 
Try clear cache and hard refresh if you see a key error from reducer. Let me know if that error persist after hard refresh.
